vagrant/provision.sh

```diff
@@ -113,6 +115,8 @@ mkdir -p $EDGE_SNAP_DIRECTORY
 chmod 777 $EDGE_SNAP_DIRECTORY
 mkdir -p $EDGE_DATA_DIRECTORY
 chmod 777 $EDGE_DATA_DIRECTORY
+mkdir -p $MAESTRO_CERTS
+chmod 777 $MAESTRO_CERTS
```

Comment:
777 seems very permissive, do we really need this to be world writable?

Reply:
👍 changing to 755